code/modules/psionics/faculties/coercion.dm

```diff
@@ -300,7 +300,7 @@
 			if(!T || L == user || L.stat == DEAD || L.invisibility == INVISIBILITY_LEVEL_TWO)
 				continue
 			if(L.isSynthetic() || L.is_diona() || isvaurca(L))
-				var/obj/item/organ/internal/augment/psiaug = locate() in L
+				var/obj/item/organ/internal/augment/psiaug/psiaug = locate() in L
```

Comment:
this is the part where i question what the hell is going on with this
I'm looking at the `has_psi_aug` proc and it returns the following item: `/obj/item/organ/internal/augment/psi`
But here you're checking for `psiaug`?
Also, why aren't you using the proc you made for this?